sdk/typescript/src/cryptography/signature.ts

```diff
@@ -9,12 +9,14 @@ export type SignatureScheme = 'ED25519' | 'Secp256k1' | 'Secp256r1' | 'MultiSig'
 /**
  * Pair of signature and corresponding public key
  */
-export type SignaturePubkeyPair = {
+export type SerializeSignatureInput = {
 	signatureScheme: SignatureScheme;
 	/** Base64-encoded signature */
 	signature: Uint8Array;
+	/** @deprecated use publicKey instead */
+	pubKey?: PublicKey;
 	/** Base64-encoded public key */
-	pubKey: PublicKey;
```

Comment:
trying to reduce these types of short names where possible (as non breaking changes)